cookbooks/php/recipes/fpm.rb

```diff
@@ -102,8 +102,8 @@ class Chef::Recipe
       :app_name => app_name,
       :php_env => node.dna['environment']['framework_env'],
       :user => node["owner_name"],
-      :dbuser => node["owner_name"],
-      :dbpass => node.engineyard.environment.ssh_password,
+      :dbuser => ode.apps.detect {|app| app[:name] == app_name}.database_username,
```

Comment:
this is a typo I believe. Lines 105-106 should be:

```
:dbuser => node.engineyard.environment.apps.detect {|app| app[:name] == app_name}.database_username,
:dbpass => node.engineyard.environment.apps.detect {|app| app[:name] == app_name}.database_password,
```

Reply:
Rad, good catch!